scripts/entrypoint.sh

```diff
@@ -28,7 +28,8 @@ entrypoint.py
 cd $HOME
 generic-worker new-ed25519-keypair --file $ED25519_PRIVKEY
 chown worker $ED25519_PRIVKEY
-envsubst < $CONF_PATH/generic-worker.yml.template > $CONF_PATH/generic-worker.yml
+# envsubst < $CONF_PATH/generic-worker.yml.template > $CONF_PATH/generic-worker-ORIG.yml
```

Comment:
What's the benefit of keeping these in the file as comments?

Reply:
It makes it easier to ensure that the configs generated are identical in future testing.